roboflow/core/project.py

```diff
@@ -227,21 +226,27 @@ def __annotation_upload(self, annotation_path, image_id):
         # Return annotation response
         return annotation_response
 
-    def check_valid_image(self, image_path):
-        try:
-            img = Image.open(image_path)
-            valid = img.format in ACCEPTED_IMAGE_FORMATS
-            img.close()
-        except UnidentifiedImageError:
-            return False
+    def check_valid_image(self, image_path, is_hosted=False):
+        if not is_hosted:
+            try:
+                img = Image.open(image_path)
+                valid = img.format in ACCEPTED_IMAGE_FORMATS
+                img.close()
+            except UnidentifiedImageError:
+                return False
+        else:
+            # for hosted images check to see if the extension is in our list of valid extensions
+            if any(ext in image_path for ext in ACCEPTED_IMAGE_EXTENSIONS):
```

Comment:
this would allow a file named `jpg.exe` to pass and is also case insensitive.
i believe this should solve both of those issues:

```suggestion
            if any(image_path.lower().endswith(ext) for ext in ACCEPTED_IMAGE_EXTENSIONS):
```

it'd be nice to add some unit tests for this function to ensure that it's catching the right things... this logic could also be helpful in `image_utils` as a new function, like `check_image_format(image_path)`.

Reply:
Same as before, no need to hardcode formats here imho